### ISSUE TITLE
`colabs/scikit/w-b-k-means-clustering.ipynb` fails to run

### ISSUE THREAD

**Issue by tcapelle** [wandb/examples#117]
[This nb](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/scikit/w-b-k-means-clustering.ipynb), Produces the following error
```python
wandb: ERROR Abnormal program exit
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_init.py in init(job_type, dir, config, project, entity, reinit, tags, group, name, notes, magic, config_exclude_keys, config_include_keys, anonymous, mode, allow_val_change, resume, force, tensorboard, sync_tensorboard, monitor_gym, save_code, id, settings)
    916         try:
--> 917             run = wi.init()
    918             except_exit = wi.settings._except_exit

9 frames
Exception: The wandb backend process has shutdown

The above exception was the direct cause of the following exception:

Exception                                 Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/six.py in raise_from(value, from_value)

Exception: problem
```

The nb does not run.
- specify project, entity should be None.
- Add markdown to create a documented notebook, if not convert ->`.py`

**Comment by udaylunawat** [comment]
I am unable to replicate this issue.
Notebook seems to run well when wandb_api_key, project and entity are provided.

**Comment by tcapelle** [comment]
Closed by #260
